services/gateway/server.py

```diff
@@ -0,0 +1,46 @@
+from typing import Annotated
+from pydantic import BaseModel
+
+import grpc
+import uvicorn
+import user_pb2_grpc, user_pb2
+from decouple import config
+from fastapi import Depends, FastAPI, HTTPException, status, Form
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
+
+app = FastAPI()
+
+@app.get("/")
+async def root():
+    return {"message":"Hello World!"}
+
+def authenticate_user(username, hashed_password):
+    with grpc.insecure_channel("localhost:" + config("MS2_PORT")) as channel:
+        stub = user_pb2_grpc.UserServiceStub(channel = channel)
+        response = stub.Login(
+            user_pb2.LoginRequest(username = username, hashed_password = hashed_password)
+        )
+        return response.auth_token
```

Comment:
The front-end should be looking for more than an auth token, missing refresh and ID.